bigtop-packages/src/common/hadoop/do-component-build

```diff
@@ -48,7 +48,28 @@ if [ $HOSTTYPE = "powerpc64le" ] ; then
                 -Dclassifier=linux-ppcle_64 -Dpackaging=exe -Dfile=/usr/src/grpc-java-1.26.0/compiler/build/exe/java_plugin/protoc-gen-grpc-java
 fi
 ## BIGTOP-2288
-
+if [ $HOSTTYPE = "x86_64" ] ; then
+        #download the POWER version of ISA-L
+        yum remove nasm -y
+        yum install yasm autoconf automake libtool -y
+        wget https://www.nasm.us/pub/nasm/releasebuilds/2.14.02/nasm-2.14.02.tar.gz
+        tar -xvf nasm-2.14.02.tar.gz
+        export NASM_HOME=`cd nasm-2.14.02; pwd`
+        cd ${NASM_HOME}
+        ./autogen.sh && ./configure
+        make
+        make install
+        cd ..
+        git clone https://github.com/intel/isa-l.git
+        export ISAL_HOME=`cd isa-l; pwd`
+        cd ${ISAL_HOME}
+        ./autogen.sh && ./configure
+        make
+        make install
+        cd ..
+        #cleanup ISA-L
+        rm -rf ${ISAL_HOME}
+fi
```

Comment:
This should be done by Puppet manifests of [bigtop_toolchain](https://github.com/apache/bigtop/tree/master/bigtop_toolchain) to avoid wasting time to build libraries every time on packaging. The clause for POWER building leveldbjni should be not here while we do not refactor it (since there is no maintainer taking care of POWER recently.)

Reply:
Okay, I'll submit the JIRA issue first and fix the restrictions on the hosttype section